src/main/resources/templates/testRules.html

```diff
@@ -159,5 +157,26 @@ <h5 class="modal-title">Aggregated Object</h5>
 	    </div>
 	  </div>
 	</div>
+	<div id="infoModal" class="modal fade" role="dialog">
+	  <div class="modal-dialog" role="document">
+	    <div class="modal-content">
+	      <div class="modal-header">
+	        <h5 class="modal-title">Usage Information</h5>
+	        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+	          <span aria-hidden="true">&times;</span>
+	        </button>
+	      </div>
+	      <div class="modal-body modal-height-fit">
+	        <p id="infoContent" class="pre"></p>
+	        <div style="text-align: center">
+	           <a href="https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/TestRulesUserGuide.md" style="text-decoration: none;">Test Rules User Guide</a>
```

Comment:
Add this to your link to open in a new tab: target="_blank" rel="noopener noreferrer"
The first part is to open in a new tab and the second is apparently to avoid potential redirections called reverse tabnabbing.
More info here: https://www.owasp.org/index.php/Reverse_Tabnabbing
The links in the navbar have vulnerable to tabnabbing though so I don't know how important that last part is. =)

Reply:
Done